tools/report/report.go

```diff
@@ -148,9 +148,9 @@ func showOnePR(prd *stats.PullRequestDetails, withDescription bool, withLabels b
 
 // ShowPRs prints a section of the report by formatting the
 // PullRequestDetails as a list.
-func ShowPRs(name string, prds []*stats.PullRequestDetails, withDescription bool, withLabels bool) {
+func ShowPRs(summarizer *enhancements.Summarizer, name string, prds []*stats.PullRequestDetails, withDescription bool, withLabels bool) error {
```

Comment:
Is there a reason to return `error`? I don't see an actual error being returned nor is it checked for in the call site.

Reply:
Ah, one iteration had this function creating the summarizer and then I moved that and didn't clean up. Fixed.